contrib/cf-remote/cf_remote/main.py

```diff
@@ -122,7 +122,9 @@ def validate_args(args):
     if args.hub:
         args.hub = file_or_comma_list(args.hub)
 
-    args.command = args.command.strip()
+    if not args.command is None:
+        args.command = args.command.strip()
+
     if not args.command:
         user_error("Invalid or missing command")
```

Comment:
This if test already checks for missing command, so can you just move the line down here instead of adding another if?